### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by marcelolagden**
Including required -AzureResourceLocation parameter and example.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Prepare the source tenant:
     | -KeyVaultName                               | Azure Key Vault instance that will store your mailbox migration application certificate/secret. | Required |
     | -CertificateName                            | Certificate name when generating or searching for certificate in key vault. | Required |
     | -CertificateSubject                         | Azure Key Vault certificate subject name, such as CN=contoso_fabrikam. | Required |
+    | -AzureResourceLocation                      | the location of the Azure Resource Group & Key Vault. | Required |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
    | -AzureResourceLocation                      | The location of the Azure resource group and key vault. | Required |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @marcelolagden, please commit my suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoeDavies-MSFT Please merge this PR. Thanks

**Pull request merged**